src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala

```diff
@@ -449,11 +451,9 @@ object DepositHandler {
     dr.setAtomStatementURI(stateIri)
     dr.setPackaging(Collections.singletonList("http://purl.org/net/sword/package/BagIt"))
     dr.setTreatment("[1] unpacking [2] verifying integrity [3] storing persistently")
```

Comment:
Is this just a String that is printed/returned somewhere, or are the `[1]... [2]... [3]...` special formatting? It's not part of this PR, but I'm just curious about it.